### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by olaughter**
- Adds more setup instructions
- Adds a task overview file linked to from the main readme
- Removes readme content that was no longer relevant

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,22 @@
...
+> [!NOTE]
+> See [pairing_interview.md](./pairing_interview.md) for details for candidates using this repo as a pairing interview.
 
-Created as an interview technical challenge. The task is to create a cli tool that can be used to search summaries of climate documents. 
-
-The cli needed to have the following functionality:
+## Functionality overview
 
 - Load & validate documents into a database at the command line
+- Write out errors to a separate file
```
Comment (diversemix) [inline README.md:10]: file / stream?

Comment (olaughter) [reply README.md:10]: That would make sense, but it only does files I'm afraid 😄

File: pairing_interview.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Candidates should ensure in advance that they have:
+- A github account
+- A local IDE setup, with python 3.10+ installed, and poetry (the repo uses v1.3.2)
+- The ability to share their screen (via Google Meet)
+
+## Context
+You will be simulating a simple ticket as a pairing exercise. This will be to add the ability to ingest JSON. Candidates should seek to add the change to the repo as a pull request. 
+
+## Task
+Context: Currently the cli requires a csv file as input, refactor it so that it can optionally support json instead. Use the json file found in the repo as confirmation: `cs/data/data.json`
```
[the remark this answers is not in the document]
Comment (olaughter) [reply pairing_interview.md:12]: Done

**Review by katybaulch — APPROVED** [review]
Cheers - looking great!

**Review by diversemix — APPROVED** [review]
Nice :t-rex:

**Pull request merged**